src/components/Sing/ScoreSequencer.vue

```diff
@@ -952,17 +1198,27 @@ const endPreview = () => {
     throw new ExhaustiveError(previewStartEditTarget);
   }
   previewMode.value = "IDLE";
+  previewNotes.value = [];
+  copiedNotesForPreview.clear();
+  edited = false;
 };
 
 const onNoteBarMouseDown = (event: MouseEvent, note: Note) => {
   if (editTarget.value !== "NOTE" || !isSelfEventTarget(event)) {
     return;
   }
+
   const mouseButton = getButton(event);
   if (mouseButton === "LEFT_BUTTON") {
+    if (isOnCommandOrCtrlKeyDown(event)) {
+      // CtrlまたはCommandキーが押されている場合、ノートを追加選択
+      void store.actions.SELECT_NOTES({ noteIds: [note.id] });
+    } else if (!selectedNoteIds.value.has(note.id)) {
+      // クリックしたノートが選択の場合、そのノートのみを選択
+      void selectOnlyThis(note);
```

Comment:
ここのコメントはこうかも。
```suggestion
      // クリックしたノートが未選択の場合、そのノートのみを選択
      void selectOnlyThis(note);
```

また、右クリックしたときにノートが選択されなくなってるかもです！（意図した変更でしょうか？）